ethdb/prune/storage_mode.go

```diff
@@ -148,38 +168,54 @@ func (p Distance) PruneTo(stageHead uint64) uint64 {
 	return stageHead - uint64(p)
 }
 
+// Before number after which keep in DB
+type Before uint64
+
+func (b Before) Enabled() bool { return b > 0 }
+func (b Before) toValue() uint64 { return uint64(b) }
+func (b Before) useDefaultValue() bool { return uint64(b) == 0 }
+func (b Before) dbType() []byte { return kv.PruneHistoryTypeBefore }
+
+func (b Before) PruneTo(uint64) uint64 {
```

Comment:
please add tests for PruneTo method. Because it's very dangerous functionality - can lead to whole db drop. Need test that it handles `uint underflow`, `uint overflow`.

Reply:
Changes in https://github.com/ledgerwatch/erigon-lib/pull/76

Reply:
Tests are added. Also I would like to add tests with `before` in `eth/stagedsync/stage_execute_test.go` 
And maybe also in `turbo/stages/blockchain_test.go`?